server/conf/i18n/messages

```diff
@@ -563,10 +563,21 @@ validation.tooManySelections.v2=Please select at most {0}.
 label.firstName=First name
 label.lastName=Last name
 label.middleName=Middle name
+label.suffix=Suffix
 
 placeholder.firstName=First name
 placeholder.lastName=Last name
 placeholder.middleName=Middle name
+placeholder.suffix=Suffix
+
+# options for suffix dropdown
```

Comment:
**suggestion (blocking):** I think it would be clearer for translators to know what kind of suffix this is.  Could you maybe say "options for the name suffix dropdown"?

Reply:
yeah that makes sense!